Installing-and-Configuring/install-os.md

```diff
@@ -0,0 +1,326 @@
+---
+layout: default
+title: Step 4 - Installing Qubes and other OSes
+permalink: /InstallingOS/
+nav_order: 8
+parent: Installing and configuring
+---
+
+<!-- markdownlint-disable MD033 -->
+<details open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+1. TOC
+{:toc}
+</details>
+<!-- markdownlint-enable MD033 -->
+
+Generic OS Installation
+===
+
+1. Insert OS installation media into one of the USB3 ports (on the left side)
+The Heads boot process supports standard OS bootable media (where the USB drive
+ contains the installation media which as created using `dd` or `unetbootin`
+ etc.) as well as booting directly from verified ISOs on a plain old partition.
+For example, if the USB drive has a single partition, you can put the ISO image
+ along with a trusted signature in the root directory:
```

Comment:
This is true only for detached signed distros for which a distro public key is present under https://github.com/osresearch/heads/tree/master/initrd/etc/distro/keys

Note issuehttps://github.com/osresearch/heads/issues/785 where Fedora is not supported anymore since not using gpg detached signatures to validate both integrity and signature, they rely on checkums and busybox is not compliant.